AssetMappr/presentation/showMap.py

```diff
@@ -37,7 +37,8 @@ def showMap(master_categories):
         html.Br(),
 
         # Container with the actual map, generated from a linked callback
-        dcc.Graph(id='graph', config={'displayModeBar': True, 'scrollZoom': True},
-                  style={'background': '#00FC87', 'height': '70vh', 'width': '100vh'}),
+        dcc.Graph(id='graph', config={'displayModeBar': True, 'scrollZoom': True}, style={
+                  'background': '#00FC87', 'height': '70vh', 'width': '100vh'})
+        #   ,style={'background': '#00FC87', 'height': '70vh', 'width': '100vh'}),
```

Comment:
You can remove commented out code that we're not using - keeps the file cleaner and easier to read